plugins/native_dio_adapter/pubspec.yaml

```
@@ -21,9 +21,9 @@ dependencies:
     sdk: flutter
 
   dio: ^5.4.0
-  cupertino_http: '>=1.0.0 <3.0.0'
-  cronet_http: '>=0.4.0 <2.0.0'
-  http: ^1.0.0
+  cupertino_http: '>=2.3.0 <3.0.0'
+  cronet_http: '>=1.5.0 <2.0.0'
```

These are inefficient, use:

```suggestion
  cupertino_http: ^2.3.0
  cronet_http: ^1.5.0
```